lib/views/backend_v1.2/spree/admin/payments/source_forms/_paypal_braintree.html.erb

```diff
@@ -0,0 +1,16 @@
+<% id = payment_method.id %>
+
+<fieldset class="no-border-bottom braintree-hosted-fields" data-braintree-hosted-fields data-id="braintree" data-payment-method-id="<%= id %>">
```

Comment:
On discussion, we agreed to leave this one as is. We anticipate that support for Solidus v1.2 may be dropped before too long, at which point it goes away.